apps/api/routers/digest.py

```
@@ -0,0 +1,88 @@
+"""Digest API Router for MVP."""
+
+import logging
+from typing import Optional
+from fastapi import APIRouter, HTTPException, Depends
```

Ruff (0.11.9)

`fastapi.HTTPException` imported but unused

Remove unused import: `fastapi.HTTPException`

(F401)